index.html

```diff
@@ -0,0 +1,451 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <div class="menu-desktop">
+        <ul class="desktop-links">
+          <li class="menu-links"><a href="#projects">Portfolio</a></li>
+          <li class="menu-links"><a href="#about-me">About</a></li>
+          <li class="menu-links"><a href="#contact">Contact</a></li>
+        </ul>
+      </div>
```

Comment:
You currently do not have the nav links showing on the desktop version of your UI. Kindly fix this bug in order to make it easy for the user to navigate and generally have a great experience on your web page.